### PR TITLE
add 2023 draft charter

### PR DESCRIPTION
**Pull request by siusin**
@johanneswilm 
please ask the group to review this draft charter, especially:

- the expected completion time;
- if the Group intends to move these specs to REC;
- any update on the Other Deliverables session?

I'll create a diff once this PR is merged.

### CODE REVIEW

**Review by johanneswilm — APPROVED** [review]

File: charter-drafts/editing-2023.html [hunk 1]
```diff
@@ -0,0 +1,582 @@
...
+              <p class="milestone"><b>Expected completion:</b> <abbr title="Candidate Recommendation">CR</abbr> Q3 2024 - Recommendation Q3 2025.</p>
+              <p><b>Adopted Draft:</b> Clipboard API and events, <a href="https://www.w3.org/TR/2023/WD-clipboard-apis-20230516/">https://www.w3.org/TR/2023/WD-clipboard-apis-20230516/</a>, 16 May 2023.
+              <p><b>Exclusion Draft:</b> Clipboard Operations for the Web 1.0: Copy, Paste, Drag and Drop. <a href='https://www.w3.org/TR/2006/WD-clipboard-apis-20061115/'>https://www.w3.org/TR/2006/WD-clipboard-apis-20061115/</a>, 15 November 2006. <a href="http://lists.w3.org/Archives/Member/member-webapi/2006Nov/0009.html">Exclusion period</a> began on 2006-11-16; ended on 2007-04-15.</p>
+              <p><b>Exclusion Draft Charter:</b> <a href='http://www.w3.org/2006/webapi/admin/charter'>http://www.w3.org/2006/webapi/admin/charter</a>.</p>
+            </dd>
+
+            <dt id="11709" class="spec">Selection API</dt>
+            <dd>
+              <p>The EditContext is a new API that allows authors to more directly participate in the text input process.</p>
```
Comment (dandclark) [inline charter-drafts/editing-2023.html]: Possible copy/paste issue, this line should talk about Selection API.

Comment (siusin) [reply charter-drafts/editing-2023.html]: Nice catch! Thank you @dandclark .

**Comment by johanneswilm** [comment]
See the proposal rendered here: https://htmlpreview.github.io/?https://github.com/w3c/editing/blob/gh-pages/charter-drafts/editing-2023.html

**Comment by siusin** [comment]
> See the proposal rendered here: https://htmlpreview.github.io/?https://github.com/w3c/editing/blob/gh-pages/charter-drafts/editing-2023.html

Thanks! 

Here's a Diff with the 2021 Charter ->
https://services.w3.org/htmldiff?doc1=https%3A%2F%2Fwww.w3.org%2F2021%2F06%2Fweb-editing-wg-charter.html&doc2=https%3A%2F%2Fw3c.github.io%2Fediting%2Fcharter-drafts%2Fediting-2023.html

**Pull request merged**